### PR TITLE
DX-1414: Add dealer app badge

### PR DESCRIPTION
**Pull request by cormickjbrowne**

### CODE REVIEW

**Comment by cormickjbrowne** [comment]
Blocks: https://github.com/1stdibs/1stdibs-admin-v2/pull/4662

**Comment by jrdrg** [comment]
@cormickjbrowne please fix conflicts

File: src/1stdibs-icon.svg [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<svg version="1.1" class="dibsvg-svg dibsvg-1stdibs-icon" viewBox="0 90 612 612" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```
Comment (nameofname) [inline src/1stdibs-icon.svg]: @cormickjbrowne please add the new icon here (the one with the square in the back) and update the name of the icon to "logo-mobile"

**Comment by nameofname** [comment]
@cormickjbrowne everything looking OK - if you get a chance go back and add a style rule to the sample page so that other users can see the mobile logo correctly

**Pull request merged**